cpp/CMakeLists.txt

```diff
@@ -262,9 +265,9 @@ if(BUILD_CUML_CPP_LIBRARY)
 
   ExternalProject_Add(faiss
     SOURCE_DIR "${FAISS_DIR}"
-    CONFIGURE_COMMAND LIBS=-pthread CPPFLAGS=-w LDFLAGS=-L${CMAKE_INSTALL_PREFIX}/lib ${FAISS_DIR}/configure --prefix=${CMAKE_CURRENT_BINARY_DIR}/faiss --with-blas=${BLAS_LIBRARIES} --with-cuda=${CUDA_TOOLKIT_ROOT_DIR} --with-cuda-arch=${FAISS_GPU_ARCHS}
+    CONFIGURE_COMMAND LIBS=-pthread CPPFLAGS=-w LDFLAGS=-L${CMAKE_INSTALL_PREFIX}/lib ${FAISS_DIR}/configure --prefix=${CMAKE_CURRENT_BINARY_DIR}/faiss --with-blas=${BLAS_LIBRARIES} --with-cuda=${CUDA_TOOLKIT_ROOT_DIR} --with-cuda-arch=${FAISS_GPU_ARCHS} -v
     PREFIX ${CMAKE_CURRENT_BINARY_DIR}/faissbuild/
-    BUILD_COMMAND $(MAKE) -j
+    BUILD_COMMAND $(MAKE) -j${PARALLEL_LEVEL} VERBOSE=1
```

Comment:
do you want to keep verbose=1 on or was that for debugging?

Reply:
Yup. Preparing a cmake follow up for controlling verbosity for developers (both treelite and Faiss included), but for CI the more prints the better to avoid potential timeouts.